### PR TITLE
[CIAPP] Add doc about how to configure CI Visibility in Jenkins via Groovy scripts

### PR DESCRIPTION
**Pull request by drodriguezhdez**
### What does this PR do?

This PR adds how to configure CI Visibility in Jenkins using Groovy (via scripts)

### Preview

https://docs-staging.datadoghq.com/drodriguezhdez/jenkins_config/continuous_integration/setup_pipelines/jenkins/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by edrevo** [comment]
The rendering is very weird on my machine:

Screenshot 2021-09-14 at 14 50 10

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 1]
```diff
@@ -41,15 +44,94 @@ Install and enable the [Datadog Jenkins plugin][3] v3.1.0 or newer:
 7. (Optional) Configure your CI Instance name.
 8. Check the connectivity with the Datadog Agent.
 9. Save your configuration.
-10. To verify that CI Visibility is enabled, go to `Jenkins Log` and search for:
+
+{{< img src="ci/ci-jenkins-plugin-config.png" alt="Datadog Plugin configuration for Jenkins" style="width:100%;">}}
+{{% /tab %}}
+{{% tab "Using configuration-as-code" %}}
+If your Jenkins instance uses the [Jenkins `configuration-as-code` plugin][1], follow the next instructions:
```
Comment (edrevo) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:51]: Opinionated, but I feel like the link here is too long. Maybe just link "configuration-as-code" or "configuration-as-code plugin"?

**Review by edrevo — APPROVED** [review]

File: content/en/continuous_integration/setup_pipelines/jenkins.md (hunk 1 above)
Comment (drodriguezhdez) [reply content/en/continuous_integration/setup_pipelines/jenkins.md]: changed

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 2]
```diff
@@ -41,15 +44,94 @@ Install and enable the [Datadog Jenkins plugin][3] v3.1.0 or newer:
 7. (Optional) Configure your CI Instance name.
 8. Check the connectivity with the Datadog Agent.
 9. Save your configuration.
-10. To verify that CI Visibility is enabled, go to `Jenkins Log` and search for:
+
+{{< img src="ci/ci-jenkins-plugin-config.png" alt="Datadog Plugin configuration for Jenkins" style="width:100%;">}}
+{{% /tab %}}
+{{% tab "Using configuration-as-code" %}}
+If your Jenkins instance uses the Jenkins [`configuration-as-code`][1] plugin, follow the next instructions:
```
Comment (kayayarai) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:51]:
```suggestion
If your Jenkins instance uses the Jenkins [`configuration-as-code`][1] plugin:
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 3]
```diff
@@ -41,15 +44,94 @@ Install and enable the [Datadog Jenkins plugin][3] v3.1.0 or newer:
...
+
+1. Create or modify the configuration yaml adding the entry for `datadogGlobalConfiguration`
```
Comment (kayayarai) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:53]:
```suggestion
1. Create or modify the configuration YAML by adding an entry for `datadogGlobalConfiguration`:
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 4]
```diff
@@ -86,6 +171,76 @@ Second, enable job log collection on the Datadog Plugin:
 6. Click on `Enable Log Collection` checkbox to activate it.
 7. Check the connectivity with the Datadog Agent.
 8. Save your configuration.
+{{% /tab %}}
+{{% tab "Using configuration-as-code" %}}
+If your Jenkins instance uses the Jenkins [`configuration-as-code`][1] plugin, follow the next instructions:
```
Comment (kayayarai) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:176]:
```suggestion
If your Jenkins instance uses the Jenkins [`configuration-as-code`][1] plugin:
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 5]
```diff
@@ -86,6 +171,76 @@ Second, enable job log collection on the Datadog Plugin:
...
+
+1. Modify the configuration yaml for the entry `datadogGlobalConfiguration`
```
Comment (kayayarai) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:178]:
```suggestion
1. Modify the configuration YAML for the entry `datadogGlobalConfiguration`:
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 6]
```diff
@@ -86,6 +171,76 @@ Second, enable job log collection on the Datadog Plugin:
...
+// Enable log collection
+d.setCollectBuildLogs(true)
+
+// Save config
+d.save()
+```
+{{% /tab %}}
+{{% tab "Using Environment Variables" %}}
+
+1. Set the following environment variables in your Jenkins instance machine:
```
Comment (kayayarai) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:226]:
```suggestion
1. Set the following environment variables on your Jenkins instance machine:
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 7]
```diff
@@ -41,15 +44,94 @@ Install and enable the [Datadog Jenkins plugin][3] v3.1.0 or newer:
...
+// (Optional) Configure your CI Instance name
+d.setCiInstanceName("jenkins")
+
+// Save config
+d.save()
+```
+{{% /tab %}}
+{{% tab "Using Environment Variables" %}}
+
+1. Set the following environment variables in your Jenkins instance machine:
```
Comment (kayayarai) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:106]:
```suggestion
1. Set the following environment variables on your Jenkins instance machine:
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

**Review by kayayarai — APPROVED** [review]

**Pull request merged**